index.html

```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="utf-8">
+	    <meta name="viewport" content="width=device-width, initial-scale=1">
+        <title>Hello Microverse</title>
+    </head>
+    <body>
+        <h1>Hello Microverse!</h1>
+    </body>
+</html>
```

Comment:
All looks wholesome to me, Except the end of the line!
```suggestion
</html>

```

Reply:
Oh my goodness! Thanks for the review!